api/host.py

```
@@ -110,6 +110,10 @@ def _add_host(input_host):
             "host",
         )
 
+    # RHCLOUD-5593
+    # Remove tags from host object here to keep from creating/updating tags from REST
+    input_host.tags = {}
+
```

🤔 should we return an error here if tags are provided? if someone uses this API before we have a chance to remove it they would be surprised to learn that tags are being silently ignored.

---

@shdunning @ryandillinfelton I think at this point we should make the change as little breaking as possible i.e. keep the formats and just change the behavior. To deal with the silent bit we can add log/metric/alert for when the tags are actually ignored.